hello-vertx/Dockerfile

```diff
@@ -6,7 +6,12 @@
 #   docker run -t -i -p 8080:8080 folio-hello-module
 ###
 
-FROM folioci/alpine-jre-openjdk11:latest
+FROM folioci/alpine-jre-openjdk17:latest
```

Comment:
Curious about the rationale for this change. If the project as a whole is moving to 17 that would be news to me. Or is it that you're wanting to keep the sample modules ahead of the "official version".